pkg/meta/base.go

```diff
@@ -1280,7 +1280,10 @@ func (m *baseMeta) Access(ctx Context, inode Ino, mmask uint8, attr *Attr) sysca
 		}
 	}
 
-	if attr.AccessACL != aclAPI.None {
+	// ref: https://github.com/torvalds/linux/blob/master/fs/namei.c
+	// ci: 5fc475b749c72e0b3f3991b33a90d302f52ae746
```

Comment:
this commit is unrelated

Reply:
Yes, I looked at the wrong line commit. 
had removed it